vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
+Entando Customer Portal is a self-service application for customers to interact within your organization to administer your Entando platform and its users. You can manage users, projects and customers. You can track projects and service tickets for your application. It is a Kubernetes-native solution built on Entando’s Application Composition Platform.
+
+Key Features:
+
+* Customizable service ticketing and tracking system with Jira Service management
+* Role-Based Access Control (RBAC)  Entando Identity Management System
+* Integrated User, customer, project and subscription management 
+
+This tutorial covers 
+1. [Installation](#installaton)
+2. [Configuration for Administrators](#configuration)
+3. [A brief User Reference](#user-refernce)
+
+# Prerequisites
+* A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/)for more information.
+* Use the Entando CLI to verify that all dependencies are installed:
+ent check-env develop
+* Docker installed and running
+
+# Installation
+## Quick Install from Entando Hub
+
+1. From the Entando Hub, click on the Customer Portal bundle from the Catalog
+2. Click Install to see a popup page with two commands. These need to be implemented in the order shown. Copy the first command and paste into your CLI. 
+3. Copy the second command and do the same. The content bundle will now be deployed.
```

Comment:
would remove "Copy the first command..." from 2. and reword 3. to something like "Run the first, then second, command from the CLI."